### ISSUE TITLE
EmojiSearchApp Sample QuickJS crashes in iOS simulator - Redwood version 0.10.0

### ISSUE THREAD

**Issue by niqo01** [cashapp/redwood#1984]
```
Assertion failed: (list_empty(&rt->gc_obj_list)), function JS_FreeRuntime, file quickjs.c, line 1985.
```
STR:
1- Git clone redwood source code and checkout tag version 0.10.0
2- Open xcode `open samples/emoji-search/ios-uikit/EmojiSearchApp.xcodeproj`
3- Build and deploy on simulator
4- the app will crash with the error above.

iOS simulator: iOS 17.4 (21E213) Model: iPhone 15 Pro

I had no issues in Redwood version 0.9.0

**Comment by JakeWharton** [comment]
This was fixed by #1949. Normally I'd do a patch release, but frankly we don't think anyone but us is really using this tool. If that's not that case, and you are using it, I'd be happy to make a new release this week.

**Comment by niqo01** [comment]
@JakeWharton I am considering Redwood for new project but I am still just at the evaluation phase. A patch release would be appreciated but I am not in a hurry, I can wait the next release cycle.
